nss-day-cohort-41/tabloid-fullstack-elvis_spotters
# Create comment jh

# Description
Comment back end code & front end code
- adds full crud to comments with ui for each function
 resolves tickets #19, #20, #21, #22 

## Type of change

# Testing Instructions
- run code in ide's of your choice
- log in using account in database
- navigate to posts, post details, comments
- a list of comments will appear (if that post has comments)
- add post buttons are both at the bottom of the post details page and at the top of the comments list
- fill out form and hit submit, user returned to comment list
- since you just filled out the form the new comment will have an edit button
- currently all users have admin privileges so each comment has a working delete button



# Checklist:
